templates/repo/issue/view_content/update_branch_by_merge.tmpl

```diff
@@ -1,9 +1,9 @@
 {{if and (gt $.Issue.PullRequest.CommitsBehind 0) (not $.Issue.IsClosed) (not $.Issue.PullRequest.IsChecking) (not $.IsPullFilesConflicted) (not $.IsPullRequestBroken)}}
 	<div class="ui divider"></div>
 	<div class="item item-section">
-		<div class="item-section-left">
-			<i class="icon icon-octicon">{{svg "octicon-alert"}}</i>
-			{{$.locale.Tr "repo.pulls.outdated_with_base_branch"}}
+		<div class="item-section-left gt-df gt-ac">
+			<i class="icon icon-octicon gt-df gt-ac">{{svg "octicon-alert"}}</i>
```

Comment:
```suggestion
			<i class="icon icon-octicon>{{svg "octicon-alert"}}</i>
```

This is likely unnecessary given as the `<i>` should not have any space inside in which to center, e.g. it wraps the `<svg>` exactly.